lib/ansible/modules/utilities/logic/wait_for.py

```diff
@@ -447,14 +446,7 @@ def _create_connection(host, port, connect_timeout):
     Returns:
         Socket object
     """
-    if sys.version_info < (2, 6):
-        (family, _) = (_convert_host_to_ip(host))[0]
-        connect_socket = socket.socket(family, socket.SOCK_STREAM)
-        connect_socket.settimeout(connect_timeout)
-        connect_socket.connect((host, port))
-    else:
-        connect_socket = socket.create_connection((host, port), connect_timeout)
-    return connect_socket
+    return socket.create_connection((host, port), connect_timeout)
```

Comment:
This function can be removed altogether. Change calls to `_create_connection()` to call `socket.create_connection()` directly.